specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2023-01-01/examples/ElasticSans_ListByResourceGroup_MaximumSet_Gen.json

```diff
@@ -30,32 +30,33 @@
               "privateEndpointConnections": [
                 {
                   "properties": {
-                    "provisioningState": "Succeeded",
+                    "provisioningState": "Invalid",
                     "privateEndpoint": {
-                      "id": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/privateEndpoints/{privateEndpointName}"
+                      "id": "ehxmltubeltzmgcqxocakaansat"
                     },
                     "privateLinkServiceConnectionState": {
                       "status": "Pending",
-                      "description": "Auto-Approved",
-                      "actionsRequired": "None"
+                      "description": "nahklgxicbqjbbvcdrkljqdhprruys",
+                      "actionsRequired": "sairafcqpvucoy"
                     },
                     "groupIds": [
-                      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ElasticSan/elasticSans/{elasticSanName}/volumegroups/{volumeGroupName}"
+                      "bolviufgqnyid"
                     ]
                   },
-                  "id": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ElasticSan/elasticSans/{elasticSanName}/privateEndpointConnections/{privateEndpointConnectionName}",
-                  "name": "{privateEndpointConnectionName}",
-                  "type": "Microsoft.ElasticSan/elasticSans/privateEndpointConnections",
+                  "id": "opcjchensdf",
+                  "name": "gewxykc",
```

Comment:
Looks like the properties in the examples got changed to random characters again. 
Is it expected to be updated to meaningful values later?